.github/workflows/manual-deploy-ten-gateway-frontend.yml

```diff
@@ -26,46 +26,54 @@ jobs:
       matrix:
         include:
           - testnet_type: 'dev-testnet'
-            GATEWAY_API_URL: 'https://dev-testnet.ten.xyz'
+            GATEWAY_API_URL: 'https://obscurogateway-dev-testnet.uksouth.cloudapp.azure.com'
           - testnet_type: 'uat-testnet'
-            GATEWAY_API_URL: 'https://uat-testnet.ten.xyz'
+            GATEWAY_API_URL: 'https://obscurogateway-uat-testnet.uksouth.cloudapp.azure.com'
           - testnet_type: 'sepolia-testnet'
-            GATEWAY_API_URL: 'https://testnet.ten.xyz'
+            GATEWAY_API_URL: 'https://obscurogateway-sepolia-testnet.uksouth.cloudapp.azure.com'
     steps:
       - name: 'Print GitHub variables'
+        if: matrix.testnet_type == github.event.inputs.testnet_type
```

Comment:
You should be able to reduce the matrix condition checks by having a top level step - cause each matrix will iterate overall the steps then to the next matrix:
```      
        - name: 'Check if selected testnet matches matrix testnet'
        if: matrix.testnet_type != github.event.inputs.testnet_type
        run: exit 0
```

Reply:
nice one, thanks 🙏 
fixed